sys/test.txt

```diff
@@ -187,3 +187,12 @@ syz_end_var_struct {
 	f1	const[0x42, int32be]
 	f2	flags[syz_end_flags, int64be]
 } [packed]
+
+# Const buffer type
+
+syz_test$bufconst(a0 ptr[in, syz_bufconst])
+
+syz_bufconst {
+	f0	bufconst["wlan0"]
```

Comment:
Where do you want to use this in real descriptions? I am asking because layout of the struct may be unexpected. I know some cases where we would like to pass an embed string constant, but size of the underlying buffer != size of the string. For example, struct contains char name[64], and we want to pass string "something_short" in name. If we write just bufconst["something_short"], its size is not equal to 64 as we want, instead it is dictated by the length of the particular string value.